Makefile

```diff
@@ -65,12 +101,21 @@ $(call target_title, "Building $(1) Image") \
 	--cache-from ${FULL_IMAGE_NAME_PREFIX}/$(1):$${__version__} $${ci_cache:-} -f $(3) $(4)
 endef
 
+# Command: build-api-image
+# Description: Build API image using the build_image method.
+# Example: make build-api-image
```

Comment:
Rather than repeat it lots, if we're giving examples of how to run it directly I assume we see this as something that can be run from the command line.

If that's the case, we should add the `##` comment so that it appears when you run `make help`

If it's not (and I think there are examples here that won't work if run directly) then we should not give examples.